### PR TITLE
Restyle highscores and extra tests

### PR DESCRIPTION
**Pull request by daanwerf**
I have restyled the highscore class into a singleton class. On top of that the highscore class now has 98.9% test coverage. Have a nice day.

### CODE REVIEW

**Comment by MichielvdBerg** [comment]
Nice Coverage and restyle, it only has conflicts with the master

**Comment by SytzeAndr** [comment]
please fix the conflicts

File: src/main/java/lasersharks/gui/LevelGUI.java [hunk 1]
```diff
@@ -75,6 +75,7 @@
   private Media media;
   private MediaPlayer mediaPlayer;
   private ImageView sharkImage;
+  private Highscores highscores = Highscores.getInstance();
```
Comment (Exclaminator) [inline src/main/java/lasersharks/gui/LevelGUI.java]: This variable should be instantiated in the start method

File: src/test/java/lasersharks/HighscoresTest.java [hunk 2]
```diff
@@ -27,6 +28,7 @@
   private static final String INPUT_FILE = "highscoresTestFile";
   private ArrayList<String> list = new ArrayList<String>();
   private final int testSize = 5;
+  private Highscores highscores = new Highscores();
```
Comment (Exclaminator) [inline src/test/java/lasersharks/HighscoresTest.java]: Ought to be instatiated prefb. setUp class

**Comment by Exclaminator** [comment]
Please fix merge conflicts so this can be pulled.

**Comment by spiderbiggen** [comment]
this branch should now be able to branch

**Comment by MichielvdBerg** [comment]
Checked

**Pull request merged**